packages/polaris-viz/src/components/LineChart/Chart.tsx

```diff
@@ -234,12 +227,8 @@ export function Chart({
       });
 
       return {
-        x:
-          xScale(activeIndex) +
-          chartXPosition +
-          parseInt(selectedTheme.chartContainer.padding, 10),
-        y: svgY,
-        position: TOOLTIP_POSITION,
```

Comment:
I think we'll have to go back to using the active index position instead of the mouse position, I just noticed the tooltip position is wrong when i use tab to focus between the points:
![image](https://github.com/Shopify/polaris-viz/assets/30587540/7a2257c2-8eef-4b12-8957-206e56ffd727)

Reply:
This case shouldn't be hit when using the keyboard. It will use the else on line 246.

I think we need to add the chart bounds like we do in the BarChart.